src/implementations/fetch_issues_from_github.rb

```diff
@@ -0,0 +1,160 @@
+# frozen_string_literal: true
+
+require 'bas/bot/base'
+require 'bas/shared_storage/postgres'
+require 'bas/utils/github/octokit_client'
+require_relative '../../src/utils/warehouse/github/issues_format'
+
+module Implementation
+  ##
+  # Implementation::FetchIssuesFromGithub
+  #
+  # This class implements a bot that fetches issues from all repositories in a GitHub organization
+  # and saves them into shared storage (e.g., PostgresDB).
+  #
+  # <b>Usage Example</b>
+  #
+  #   read_options = {
+  #     connection: Config::CONNECTION,
+  #     db_table: 'warehouse_sync',
+  #     avoid_process: true,
+  #     where: 'archived=$1 AND tag=$2 ORDER BY inserted_at DESC',
+  #     params: [false, 'FetchIssuesFromGithub']
+  #   }
+  #
+  #   write_options = {
+  #     connection: Config::CONNECTION,
+  #     db_table: 'warehouse_sync',
+  #     tag: 'FetchIssuesFromGithub'
+  #   }
+  #
+  #   options = {
+  #     private_pem: Config::GITHUB_PRIVATE_PEM,
+  #     app_id: Config::GITHUB_APP_ID,
+  #     organization: Config::KOMMITERS_ORGANIZATION
+  #   }
+  #
+  #   shared_storage = Bas::SharedStorage::Postgres.new({ read_options:, write_options: })
+  #
+  #   Implementation::FetchIssuesFromGithub.new(options, shared_storage).execute
+  #
+  class FetchIssuesFromGithub < Bas::Bot::Base
+    PER_PAGE = 100
+
+    # Orchestrates the fetching and formatting of records from GitHub.
+    #
+    def process
+      client_response = initialize_client
+      return client_response if client_response[:error]
```

Comment:
This could be a problem because the BAS gem is not expecting this kind of response. Can you format the error to be something consistent with the BAS?